arch/xtensa/src/lx6/Toolchain.defs

```diff
@@ -55,8 +59,15 @@ else
   MAXOPTIMIZATION += -fomit-frame-pointer
 endif
 
-ifeq ($(CONFIG_MM_KASAN),y)
-  ARCHCPUFLAGS += -fsanitize=kernel-address
+ARCHCFLAGS = -mlongcalls
+ARCHCXXFLAGS = -mlongcalls
```

Comment:
So why `-mlongcalls` was not needed before?

Reply:
It was already required, it was declared in `ARCHCFLAGS` of each Xtensa-based board.
And `ARCHCFLAGS` was expanded into `ARCHCXXFLAGS`.

Reply:
@gustavonihei look like this more like ARCHCPUFLAGS flag?

Reply:
Indeed, I'll fix it! Thanks